railgun/Makefile

```diff
@@ -1,3 +1,9 @@
+.PHONY: clean
+clean:
+	@rm -rf testbin/
+	@rm -rf bin/*
+	@rm -f coverage.out
```

Comment:
```suggestion
	@rm -rfv testbin/ bin/* coverage.out
```

Reply:
Wait, `-v` conflicts with the `@`.
Please either make this silent or verbose - your call here.

Reply:
Personally, I don't mind the verbosity so given the choice I'll keep it, thank you :bow: